jetbrains-core/src/software/aws/toolkits/jetbrains/services/lambda/resources/LambdaResources.kt

```diff
@@ -13,5 +13,5 @@ object LambdaResources {
         listFunctionsPaginator().functions().filterNotNull().toList()
     }
 
-    fun function(arn: String) = LIST_FUNCTIONS.find { it.functionArn() == arn }
+    fun function(name: String) = LIST_FUNCTIONS.find { it.functionName() == name }
```

Comment:
were there are places this was used?

Reply:
![Screen Shot 2019-09-23 at 12 33 56 PM](https://user-images.githubusercontent.com/8992246/65456577-73752700-ddfe-11e9-84cb-6ef5d615c768.png)

Reply:
yup just ran that :P